packages/sdk-utilities/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "files": [],
+  "include": [],
+  "references": [
+    {
+      "path": "./tsconfig.lib.json"
+    },
+    {
+      "path": "./tsconfig.spec.json"
+    }
+  ],
+  "nx": {
+    "addTypecheckTarget": false
```

Comment:
curious about this, i'll have to look it up. its confusing to me that this is false but we do still run `typecheck`.

i'm almost curious if we should just remove it.